scripts/format_app.py

```diff
@@ -0,0 +1,70 @@
+# format_app.py
```

Comment:
Maybe this could make this more easy to execute:

```suggestion
#!/usr/bin/env python3

# format_app.py
```
(don't forget to `chmod +x` :wink: )